docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -327,18 +368,18 @@ In this section, we will:
     type: "video"
   }).then(function(response) {
     response //-> {
-	// result: {
-	//   items: [
-	//     {
-	//       id: {videoId: "ajsadfa"},
-	//       snippet: {
-	//         title: "dogs",
-	//         thumbnails: {default: {url: "https://example.com/dog.png"}}
-	//       }
-	//     }
-	//   ]
-	// }	 
-	//}
+  // result: {
+  //   items: [
+  //     {
+  //       id: {videoId: "ajsadfa"},
+  //       snippet: {
+  //         title: "dogs",
+  //         thumbnails: {default: {url: "https://example.com/dog.png"}}
+  //       }
+  //     }
+  //   ]
+  // }	 
+  //}
```

Comment:
This looks like it should be indented more.

Reply:
Fixed